app/controllers/links_controller.rb

```diff
@@ -38,6 +40,19 @@ def documentation_url(section = "", options = {})
     root_url + (section || '')
   end
 
+  # For new documentation at docs.theforeman.org
+  # options:
+  #   :section
+  #   :index    'foreman-el' (default), 'foreman-deb' or 'katello'
+  #   :chapter
+  def new_documentation_url(options)
```

Comment:
Can we use an API here with explicit parameters? At least for those that are required (like section).